lizmap/modules/lizmap/lib/Request/OGCRequest.php

```diff
@@ -185,6 +185,76 @@ protected function constructUrl()
         return Proxy::constructUrl($this->parameters(), $this->services, $url);
     }
 
+    /**
+     * Generate a string to identify the target of the HTTP request.
+     *
+     * @param array $parameters The list of HTTP parameters in the query
+     * @param int   $code       The HTTP code of the request
+     *
+     * @return string The string to identify the HTTP request, with main OGC parameters first such as MAP, SERVICE...
+     */
+    private function formatHttpErrorString($parameters, $code)
+    {
+        // Clone parameters array to perform unset without modify it
+        $params = array_merge(array(), $parameters);
+
+        // Ordered list of params to fetch first
+        $mainParamsToLog = array('map', 'repository', 'project', 'service', 'request');
+
+        $output = array();
+        foreach ($mainParamsToLog as $paramName) {
+            if (array_key_exists($paramName, $params)) {
+                $output[] = '"'.strtoupper($paramName).'" = '."'".$params[$paramName]."'";
+                unset($params[$paramName]);
+            }
+        }
+
+        $message = implode(' & ', $output);
+
+        if ($params) {
+            // It does not work in jLog
+            // $message .= '\n';
+            $message .= ' & ';
+        }
+
+        // For remaining parameters in the array
+        $output = array();
```

Comment:
Won't this will do the job ?

(`$message` set at the end, `$output`, 
 ```suggestion
         // For remaining parameters in the array
```